lib/code_corps/github/sync/sync.ex

```diff
@@ -258,11 +249,11 @@ defmodule CodeCorps.GitHub.Sync do
   end
```

Comment:
I'm not sure this abstraction is a net gain. My vote would be to remove it. The DRYness we get is not worth the loss in information and clarity.